source/standards/tracking-dependencies.html.md.erb

```diff
@@ -73,7 +73,7 @@ Also consider managed solutions where possible. For example:
 * Use [GOV.UK PaaS buildpacks][] so you do not have to monitor for Operating System (OS), runtime or programming language vulnerabilities
 * Consider managed container orchestration technology such as [AWS Fargate][] or similar so you do not have to monitor for vulnerabilities in your OS or control plane
 
-This guidance is in line with the GDS Reliability Engineering strategic principle of [use fully managed cloud services by default][].
+This guidance is in line with the GDS Reliability Engineering strategic principle of [using fully managed cloud services by default][].
```

Comment:
This bit is missing it's URL ... think you need to update the name of the link to match `using fully managed cloud services by default` on line 76